docs/core-manage-plugins.md

```diff
@@ -1,24 +1,25 @@
-Plugins are how asdf understands handling different packages.
+Plugins are how asdf knows how to handle different packages.
```

Comment:
"packages" here might be a little ambigous. It might be good to elaborate, something like "how to handle different tools, like Ruby, Elixir, Node.JS, etc..."